### DISCUSSION TITLE
nginx plus nginx-repo.crt license expired

### DISCUSSION THREAD

**Discussion by nanpingxiang** [nginx/kubernetes-ingress#6765] [General]
If NGINX Plus certificate and public key (nginx-repo.crt and nginx-repo.key files) expire, will it affect my current nginx functions?
By the way, I am using nginx plus in nginx ingress controller.

**Comment by AlexFenlon** [comment]
Hi @nanpingxiang, 

No, it will not but please check the FAQ for NGINX Plus on the F5 website [here](https://github.com/nginxinc/kubernetes-ingress/blob/f9b5ce5ed9c9201ec10012ba2f9c4b7438904656/cmd/nginx-ingress/main.go#L673)

**Reply by vepatel** [reply]
@nanpingxiang just to clarify a bit more, you won't be able to upgrade, create new images or pull a new one.
